pkg/storage/provider_abort_action_test.go

```diff
@@ -0,0 +1,272 @@
+package storage_test
+
+import (
+	"testing"
+
+	"github.com/bsv-blockchain/go-wallet-toolbox/pkg/internal/fixtures"
+	"github.com/bsv-blockchain/go-wallet-toolbox/pkg/internal/fixtures/testusers"
+	"github.com/bsv-blockchain/go-wallet-toolbox/pkg/storage/internal/testabilities"
+	"github.com/bsv-blockchain/go-wallet-toolbox/pkg/wdk"
+	"github.com/go-softwarelab/common/pkg/to"
+	"github.com/stretchr/testify/require"
+)
+
+const NumberOfDesiredUTXOs = 32
+
+func TestAbortActionSuccess(t *testing.T) {
+	// given:
+	given, cleanup := testabilities.GivenCustomStorage(t, fixtures.StorageServerPrivKey, "dbstorage_test")
+	defer cleanup()
+
+	activeStorage := given.Provider().GORM()
+	faucet := given.Faucet(activeStorage, testusers.Alice)
+	faucet.TopUp(100_000)
+
+	createResult, err := activeStorage.CreateAction(
```

Comment:
It would be greate to have something similar to
`given.ActionCreatedAndSigned(activeStorage)` 
that would work here